main.js

```diff
@@ -0,0 +1,48 @@
+export { default as HdExpandText } from './src/components/HdExpandText.vue';
+export { default as HdLazyImage } from './src/components/HdLazyImage.vue';
+export { default as HdLink } from './src/components/HdLink.vue';
+export { default as HdPager } from './src/components/HdPager.vue';
+export { default as HdResponsive } from './src/components/HdResponsive.vue';
+export { default as HdSuggestionItem } from './src/components/HdSuggestionItem.vue';
+export { default as HdTable } from './src/components/HdTable.vue';
+export { default as HdTabsMenu } from './src/components/HdTabsMenu.vue';
+export { default as HdTagsList } from './src/components/HdTagsList.vue';
+export { default as HdTimeslots } from './src/components/HdTimeslots.vue';
+export { default as HdToast } from './src/components/HdToast.vue';
+
+// Form
+export { default as HdAutocomplete } from './src/components/form/HdAutocomplete.vue';
+export { default as HdCheckbox } from './src/components/form/HdCheckbox.vue';
+export { default as HdDynamicForm } from './src/components/form/HdDynamicForm.vue';
+export { default as HdGoogleAutocomplete } from './src/components/form/HdGoogleAutocomplete.vue';
+export { default as HdInput } from './src/components/form/HdInput.vue';
+export { default as HdInputPassword } from './src/components/form/HdInputPassword.vue';
+export { default as HdPasswordConfirm } from './src/components/form/HdPasswordConfirm.vue';
+export { default as HdRadio } from './src/components/form/HdRadio.vue';
+export { default as HdRange } from './src/components/form/HdRange.vue';
+export { default as HdSelect } from './src/components/form/HdSelect.vue';
+export { default as HdSplitInput } from './src/components/form/HdSplitInput.vue';
+export { default as HdTagsSelector } from './src/components/form/HdTagsSelector.vue';
+export { default as HdTextarea } from './src/components/form/HdTextarea.vue';
+
+// Tooltip
+export { default as HdTooltip } from './src/components/tooltip/HdTooltip.vue';
+export { default as HdTooltipped } from './src/components/tooltip/HdTooltipped.vue';
+export { default as HdTooltipInstaller } from './src/components/tooltip/installer';
+
+// Gallery
+export { default as HdGallery } from './src/components/gallery/HdGallery.vue';
+export { default as HdGalleryCarousel } from './src/components/gallery/HdGalleryCarousel.vue';
+export { default as HdGalleryMedia } from './src/components/gallery/HdGalleryMedia.vue';
+export { default as HdGalleryOverlay } from './src/components/gallery/HdGalleryOverlay.vue';
+export { default as HdGalleryPlaceholder } from './src/components/gallery/HdGalleryPlaceholder.vue';
+export { default as HdGalleryTiles } from './src/components/gallery/HdGalleryTiles.vue';
+export { default as HdZoomerGallery } from './src/components/gallery/HdZoomerGallery.vue';
+
+// Notifications
+export { default as HdNotifications } from './src/components/notifications/HdNotifications.vue';
```

Comment:
Should we export services here as well? Otherwise you'd import them as `homeday-blocks/src/services/SERVICE_NAME`. What do you think?

Reply:
Could be! Gonna work on it!

Reply:
Could you check the exported services?